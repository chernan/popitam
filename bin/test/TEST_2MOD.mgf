BEGIN IONS
TITLE=VFNC[cam]ISYSPLC[cam]K Q9GZL7
PEPMASS=744.3874125
CHARGE=2+
72.0910 10.1429
72.1344 0.3333
87.0717 19.8095
120.0991 89.0953
120.2066 3.3810
121.0929 5.8571
132.0973 48.4762
133.0597 15.6190
134.0456 2.1905
136.1085 4.9048
147.1329 16.6667
183.1597 8.2381
184.0944 8.2381
185.1205 10.3810
201.1621 3.1905
211.1579 8.5238
213.0724 2.9048
219.1731 168.6667
219.2609 11.5714
220.1812 19.9524
246.1502 18.8095
246.2386 3.4762
247.1625 148.6191
247.3412 5.7143
248.1723 22.1429
252.1615 6.4286
253.1611 2.1905
254.1470 2.1429
258.0684 8.7619
261.1396 2.1905
262.1379 56.1905
263.1713 9.3810
265.1945 2.1905
270.1793 5.2381
271.1994 2.1905
272.1403 8.3810
273.0995 4.2857
274.1375 13.5238
275.1006 87.6190
276.1126 22.3333
279.1688 18.0476
280.1541 19.4762
280.2836 4.9524
282.1966 11.8095
282.3061 2.9524
287.2300 5.2381
287.3473 2.0476
289.1598 18.8095
289.4271 2.1905
290.1542 12.2381
290.2601 0.4762
291.1245 3.9048
292.1597 4.1905
297.1781 12.6667
298.1833 11.5238
298.3131 1.9048
299.1642 3.1905
303.1585 5.2857
307.1666 146.3810
308.1703 21.4286
309.1590 4.2857
314.1338 5.0476
315.2153 7.1905
316.2058 8.2381
318.1797 2.3810
326.1561 6.3810
329.2281 5.4286
330.1519 2.1905
331.1668 28.0476
332.1654 1.9048
332.2821 5.5238
338.1833 7.3810
343.1669 13.2381
344.1947 11.3810
345.1598 4.2857
348.1965 3.1905
353.1743 5.3810
357.1844 2.1905
359.1454 5.1905
360.2258 5.4286
361.2120 53.0000
361.3527 7.2381
362.1985 10.4762
367.2195 5.7143
371.0427 1.3333
371.1852 34.1429
372.1971 11.0000
372.3270 1.7143
373.2081 7.4762
374.1763 4.2857
377.1805 6.3810
378.2362 56.3333
378.4496 4.8095
379.2432 15.1429
381.2386 5.3810
386.5892 2.1905
388.2026 52.2381
389.2006 26.6667
394.1762 5.1905
396.2443 19.3333
399.2238 23.5714
400.1935 4.0000
401.1972 5.2381
402.2278 8.2857
405.1525 13.1905
406.1161 2.8095
406.2162 5.1429
412.2753 4.2857
413.2297 5.0476
416.2743 7.2857
417.2093 7.3810
420.2627 14.0476
420.4588 1.6667
421.2319 7.1429
422.1651 27.2857
423.1902 13.4762
423.2719 5.8571
426.2850 7.0476
427.2230 21.8095
428.2856 3.1905
429.2750 13.8095
430.2355 67.6667
430.3770 8.5238
431.2368 13.4762
435.2224 6.3333
440.1749 3.9048
444.2446 19.6191
444.3744 3.7619
445.2430 8.3333
446.3358 3.5238
447.2761 3.1905
448.2949 3.1429
450.1780 8.0952
451.2287 3.1905
452.1899 4.1905
453.2556 3.1905
457.2038 7.3810
458.2172 25.9524
458.3550 2.8095
459.2325 6.3333
460.2501 4.1905
473.2450 5.2381
474.2546 2.1905
475.2266 8.1905
486.2707 7.0476
487.1278 0.5238
487.2441 9.8571
488.2654 7.4286
489.2283 3.3333
490.2242 13.1429
491.2591 3.1905
493.2614 8.9524
494.3890 2.0952
495.2327 6.1905
497.1837 4.2857
498.1714 2.1905
498.3051 14.0476
499.2697 23.5714
500.2811 9.0476
504.2260 4.2857
506.2237 4.2857
507.2334 9.4762
508.2771 3.1905
512.2968 3.1905
513.3693 8.0476
514.2020 3.1905
515.3242 8.4762
517.0178 1.5238
517.1652 10.7143
517.3057 286.0953
517.5369 20.1905
517.8826 3.6190
518.3159 61.8095
518.6421 5.0476
519.2986 16.8571
519.4536 2.4286
520.2586 9.0476
521.2489 39.3333
522.2456 21.1905
523.2428 4.1905
523.7419 4.1905
524.2390 3.1905
530.2700 7.0952
530.3872 1.2381
531.2565 2.1905
535.2610 9.3810
543.3308 11.0476
544.3034 7.9048
545.2618 4.2857
548.3138 9.1905
550.2565 5.1905
559.2336 2.7619
559.3364 1.7143
563.2390 5.2381
565.3084 9.2381
566.3063 4.1905
567.3436 8.2381
568.2943 3.1905
569.3079 5.8571
572.2667 2.1905
574.3237 5.1429
577.2826 5.1905
584.3476 5.0476
586.3235 18.1905
588.3463 5.3333
589.3042 23.5238
590.2870 9.2381
593.2811 4.7143
594.7924 3.1905
595.2642 4.2857
599.3337 6.4286
604.1481 1.4762
604.3371 163.1429
604.5664 5.9048
605.3364 52.4762
606.3358 21.0476
606.8173 2.1905
607.3043 9.0476
608.2727 3.1905
611.2850 4.1905
617.3344 6.3810
621.1271 2.1905
621.3034 7.4286
633.3910 4.6190
634.3231 7.7619
635.3576 3.9524
659.3152 4.2857
661.3756 6.9048
662.1945 2.0952
662.3121 9.5714
674.2780 3.5238
676.3393 2.4286
676.4965 3.0000
680.3472 4.2857
686.2989 7.1905
687.3080 2.1905
690.3232 7.2857
690.5265 1.9524
691.2938 6.1429
704.3560 5.3333
708.3231 14.2381
709.3516 6.1905
714.3994 5.7619
721.3676 5.4286
731.3709 3.1905
735.8340 3.1905
744.0260 2.1905
744.4536 6.1905
745.4155 17.3333
745.6699 4.5238
750.2410 2.5238
750.4180 5.8571
751.3363 3.1429
753.3011 5.4762
767.2026 5.2857
767.4118 83.5238
767.9831 2.1905
768.3981 33.5714
769.0979 2.4762
769.4051 10.3333
773.4003 3.5714
793.3347 7.2857
801.4106 3.3333
819.3909 4.2857
820.3486 4.2857
821.4442 6.2857
822.3530 2.1905
836.3759 10.3333
839.1409 2.1905
839.4205 5.1905
854.2169 12.2381
854.4283 323.3810
854.7192 19.6667
854.9260 3.6667
855.0499 9.2857
855.4226 130.7619
855.8972 7.5714
856.1729 13.9048
856.4431 48.1905
857.3755 3.1905
864.4464 5.0476
881.4122 2.8571
931.4593 3.3333
949.5067 8.2381
950.4214 4.2857
967.2586 7.2381
967.5146 132.9048
967.7787 11.0476
968.5186 69.9524
968.7062 8.0476
968.9529 0.6667
969.3566 3.6190
969.5140 20.5238
1036.6182 2.1905
1037.5865 4.1905
1079.4755 2.1905
1092.6090 3.1905
1110.2574 1.4286
1110.4097 2.1905
1110.5582 3.2857
1111.4711 5.1905
1111.9474 2.1905
1112.5454 3.6190
1116.5524 2.1905
1127.5454 56.4286
1128.3638 4.3333
1128.5669 36.6667
1129.2528 2.6190
1129.5746 10.8095
1150.5327 5.4762
1223.5685 4.6667
1224.5856 7.4286
1225.5167 11.4762
1225.7324 3.2857
1241.3105 6.3810
1241.6077 56.3333
1241.9739 8.7143
1242.3627 4.0952
1242.6147 51.9524
1243.5609 26.3810
1244.6045 5.2381
1246.5846 2.1905
1371.6332 4.1905
1388.8130 1.9524
1389.6213 3.1905
1390.6581 5.2381
END IONS
