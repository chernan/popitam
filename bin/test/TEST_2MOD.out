Popitam v4.0 is now running; run parameters are NORMAL and UNKNOWN (2 allowed gaps)

________________________________________________________________________________

File processed                               : test/TEST_2MOD.mgf
Initial number of spectra                    : 1
Instrument                                   : QTOF
Min covbin                                   : 2
Wanted number of coverage                    : 9
Minimal tag lentgh                           : 3
Minimal coverage for a valuable arrangement  : 0.300000

Graph parameters                              
 - edges_type :                              : 1,2 aa
 - peptide error :                           : 2.00
 - fragment error1 :                         : 0.20
 - fragment error2 :                         : 0.30

DBs                                           
demoDB_FORWARDandDECOY                       : BASE_FORWARDandDECOY_1.0
TAXONOMY                                     : NO
AC FILTER                                    : P48668 DECOY_P48668 Q80V08 DECOY_Q80V08 P68871 DEC...  

Enzyme                                       : Trypsin
Missed cleavage number                       : 2

Modification/Mutation parameters              
up limit of PM range                         : 400.00
low limit of PM range                        : -200.00
up limit of MOD range                        : 400.00
low limit of MOD range                       : -200.00


________________________________________________________________________________

Spectrum ID              : 0
SpectrumTitle            : VFNC[cam]ISYSPLC[cam]K Q9GZL7
initPeakNb               :         326
peakNb after proc.       :          57
ParentMass raw           :   744.38741
ParentMass(M)/Charge     :  1486.77482 / 2      
NodeNb                   :     128 
EdgeNb (simple/double)   :     287 / 3356   


________________________________________________________________________________


Total number of proteins in databases                                                          :      1666
Number of proteins after filter (taxonomy, ACs...)                                             :        24
Number of peptides after filter (PM...)                                                        :       241
Number of peptides with one or more scenario                                                   :       240
Cumulated number of scenarios                                                                  :      2885

SAMPLE SIZES for statistics: 239 peptides neg

        # | score (deltaS ; pValue)     | mass        | access                    | id                   | dtbSeq / scenarios (shifts) 
--------- | --------------------------- | ----------- | ------------------------- | -------------------- | -------------------- 
 1. (  2) | 659853.403 (0.85 ; 0.0e+00) |  1372.65186 | Q9GZL7                    | WDR12_HUMAN          | VFNCISYSPLCK
          |                             |             | DECOY_Q9GZL7              | WDR12_HUMAN          | vfN*ISYSpl**         57.08    57.05 
--------- | --------------------------- | ----------- | ------------------------- | -------------------- | -------------------- 
 2. (  2) | 562242.049 (0.05 ; 0.0e+00) |  1528.75305 | Q9GZL7                    | WDR12_HUMAN          | VFNCISYSPLCKR
          |                             |             | DECOY_Q9GZL7              | WDR12_HUMAN          | vfN*ISYSpl***        57.08   -99.06 
--------- | --------------------------- | ----------- | ------------------------- | -------------------- | -------------------- 
 3. (  2) |  30368.997 (0.66 ; 1.0e+00) |  1573.84363 | P48668                    | K2C6C_HUMAN          | RGFSANSARLPGVSR
          |                             |             | DECOY_P48668              | K2C6C_HUMAN          | ---saNS*RLP*Vsr     -65.09   -22.00 
--------- | --------------------------- | ----------- | ------------------------- | -------------------- | -------------------- 
 4. (  2) |  20161.242 (0.46 ; 1.0e+00) |  1114.58704 | P48668                    | K2C6C_HUMAN          | LEGLEDALQK
          |                             |             | DECOY_P48668              | K2C6C_HUMAN          | **GLED*LQK          -23.99   396.21 
--------- | --------------------------- | ----------- | ------------------------- | -------------------- | -------------------- 
 5. (  2) |   9314.719 (1000 ; 1.0e+00) |  1106.63354 | P18621                    | RL17_HUMAN           | YLKDVTLQK
          |                             |             | DECOY_P18621              | RL17_HUMAN           | *LKD**LQK           112.06   268.11 
--------- | --------------------------- | ----------- | ------------------------- | -------------------- | -------------------- 

Processing time was:	0.820000


NEXT______________________________________________________________



RESULTS BY PROTEIN (MINIMUM PEPTIDE COVERAGE REQUIRED FOR DISPLAY IS 5): 

NO CANDIDATE PROTEIN 
